frontend/src/metabase-types/analytics/search.ts

```diff
@@ -33,16 +35,44 @@ type SearchContentType =
   | "action"
   | "indexed-entity";
 
-type SearchContext =
+type SnowplowSearchContext =
   | "search-app"
   | "search-bar"
   | "command-palette"
   | "entity-picker";
 
+// TODO: once we've finalized the new context names, we should add them to the Snowplow schema.
+// It would also be good to use an exclusion list, so that this is easier to maintain when new context are added.
+const RegisteredSearchContext = [
```

Comment:
nit: we usually use UPPER_CASE for constants like these, otherwise it looks like a type